precompiles/bank/bank.go

```diff
@@ -150,7 +153,18 @@ func (p PrecompileExecutor) send(ctx sdk.Context, caller common.Address, method
 		return nil, err
 	}
 
-	if err := p.bankKeeper.SendCoins(ctx, senderSeiAddr, receiverSeiAddr, sdk.NewCoins(sdk.NewCoin(denom, sdk.NewIntFromBigInt(amount)))); err != nil {
+	msg := &banktypes.MsgSend{
```

Comment:
do we want to change it for `sendNative` too? (`send` takes an EVM recipient, `sendNative` takes a `sei...` address recipient)

Reply:
`sendNative` uses `SendCoinsAndWei` which does not have msg server counterpart. We could refactor that first and then re-visit `sendNative`